edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/hubStart.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+###############################################################################
+# Set up EdgeHub to run as a non-root user at runtime, if allowed.
+# 
+# If this script is started as root:
+#  1. It reads the EDGEHUBUSER_ID environment variable, default UID=13623.
+#  2. If the User ID does not exist as a user, create it.
+#  3. If "StorageFolder" env variable exists, use as basepath, else use /tmp
+#     Do same for backuppath
+#  4. If basepath/edgehub exists, make sure all files are owned by EDGEHUBUSER_ID
+#     Do same for backuppath/edgehub_backup
+#  5. Make sure /app/backup.json is writeable.
```

Comment:
This is not used in EdgeHub

Reply:
Forgot to remove the comment. It is removed in the script.